cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -97,12 +112,51 @@ func main() {
 
 				if len(unschedulablePodsToHelp) == 0 {
 					glog.V(1).Info("No unschedulable pods")
-					continue
+				} else {
+					scaledUp, err := ScaleUp(unschedulablePodsToHelp, nodes, migConfigs, gceManager, kubeClient, predicateChecker)
+					if err != nil {
```

Comment:
When err is not nil? Don't we want to bump lastScaleUpTime here as well to avoid the situation we failed to resize mig for 10 minutes (and we wanted to do it) so that we started scaling down?